pages/1.11/release-notes/1.11.0-rc4/index.md

```diff
@@ -89,32 +89,30 @@ Please try out the new features and updated data services. Provide any feedback
 - Upgrades from 1.10.5 to 1.11.0-rc4 are supported, but not recommended.
 - Upgrades from 1.11.0-rc1 to 1.11.0-rc4 are _not supported_ in 1.11 Release Candidate 4.
 
-# <a name="fixed-issues"></a>Improvements and Major Issues Fixed since 1.11.0 Release Candidate 1
-- DCOS-19500  - Fix cosmos/service/update failure when Marathon is deploying
+# <a name="fixed-issues"></a>Improvements and Major Issues Fixed in 1.11.0 Release Candidate 4
+- COPS-2201 - Fix `dcos-diagnostics` crashing after running gen_resolvconf.py.
+- COPS-2465 - License reporting is inconsistent. [enterprise type="inline" size="small" /]
+- DCOS-19500 - Fix cosmos/service/update failure when Marathon is deploying.
+- DCOS-20064 - Verify Azure and AWS advanced templates work with the licensing parameter. [enterprise type="inline" size="small" /]
+- DCOS-20343 - Fix UI loading screen.
+- DCOS-20396 - Test that bootstrap regenerates cert when old cert does not match new CSR.
+- DCOS-20485 - Adjust implementation to address performance issues.
+- DCOS-20492 - Cluster sometimes comes up without license when using AWS CloudFormation templates. [enterprise type="inline" size="small" /]
+- DCOS-20515 - Make licensing version-aware to prevent upgrade failures. [enterprise type="inline" size="small" /]
+- DCOS-20569 - Include only public attributes of license on diagnostics bundle. [enterprise type="inline" size="small" /]
+- DCOS-20628 - `dcos-vault`: fix deadlock in go-zookeeper library.
+- DCOS-20676 - `csidevices`: fix lsblk parsing of empty string bool values.
+- DCOS-20679 - Make licensing handle a cluster license renewal for same major version during a config upgrade. [enterprise type="inline" size="small" /]
+- DCOS-20772 - Limit znode creation.
```

Comment:
`Limit znode creation` - line should be replaced with a meaningful changelog line.